### DISCUSSION TITLE
Difference of layout

### DISCUSSION THREAD

**Discussion by brokenby2703** [dec0dOS/zero-ui#123] [Q&A]
Hi,

I would like to know whether it's my installation that make the ZEROTIER look different from the original MY.ZEROTIER.com

This is the original:

<img width="1079" alt="Screenshot 2022-11-18 at 19 38 05" src="https://user-images.githubusercontent.com/115927083/202779596-e51791de-c975-433f-8f70-7e1828bc259c.png">

This is my:

<img width="961" alt="Screenshot 2022-11-18 at 19 38 23" src="https://user-images.githubusercontent.com/115927083/202779631-e7bde513-8c42-4a9e-91e5-bd84b822cf86.png">

Is it possible to have ZERO-ui same look and feel of the original ?

Thanks.

**Comment by dec0dOS** [comment]
Hello, @brokenby2703!

No, ZeroUI is not going to copy the UI from ZeroTier Central.
Although, there are some improvements that should be made to provide more information about the networks on the main ZeroUI page.

**Comment by brokenby2703** [comment]
Thanks a lot.

Dado

-------- Original Message --------
